src/PowerShellEditorServices/Services/DebugAdapter/BreakpointService.cs

```diff
@@ -43,6 +44,7 @@ public async Task<List<Breakpoint>> GetBreakpointsAsync()
         {
             if (BreakpointApiUtils.SupportsBreakpointApis(_editorServicesHost.CurrentRunspace))
             {
+                _editorServicesHost.Runspace.ThrowCancelledIfUnusable();
```

Comment:
So it only really applies to remote runspaces and almost never the main runspace. So we just sorta didn't have to care because it didn't really work in the first place :P

Reply:
Ahh that makes sense.